check-diff.sh

```diff
@@ -166,6 +166,13 @@ function set_environment_variables {
 		ESLINT_IGNORE="$( upsearch .eslintignore )"
 	fi
 
+	if [ -z "$STYLELINT_CONFIG" ]; then
+		STYLELINT_CONFIG="$( upsearch .stylelintrc )"
+	fi
+	if [ -z "$STYLELINT_CONFIG" ]; then
+		STYLELINT_CONFIG="$( upsearch stylelint.config.js )"
+	fi
+
```

Comment:
stylelint supports a few more config files, not sure if you want to include all possible combinations here

> Via https://stylelint.io/user-guide/configuration/
> The `.stylelintrc` file (without extension) can be in JSON or YAML format. Alternately, you can add a filename extension to designate JSON, YAML, or JS format: `.stylelintrc.json`, `.stylelintrc.yaml`, `.stylelintrc.yml`, `.stylelintrc.js`. You may want to use an extension so that your text editor can better interpret the file, and help with syntax checking and highlighting.